crates/pallet-feeds/src/lib.rs

```diff
@@ -17,29 +18,78 @@ pub mod pallet {
     pub struct Pallet<T>(_);
 
     pub type PutDataObject = Vec<u8>;
+    pub type FeedId = u64;
+
+    // TODO: make it more generic
+    #[derive(Encode, Decode, Debug, Clone, Eq, PartialEq)]
```

Comment:
After Substrate upgrade you need to also derive `scale_info::TypeInfo` similarly to other data types in the repo that are used in runtime (this is the reason of CI failure).

Reply:
yes, I had it in the original PR before doing rebase, thanks for pointing